theia-docker/Dockerfile

```diff
@@ -9,10 +9,17 @@ RUN yarn && \
     yarn cache clean
 
 FROM node:8-alpine
+# See : https://github.com/theia-ide/theia-apps/issues/34
+RUN addgroup theia && \ 
+    adduser -G theia -s /bin/sh -D theia;
+RUN chmod g+rw /home && \
+    mkdir /home/theia/project && \
+    chown theia:theia /home/theia/project;    
 RUN apk add --no-cache git openssh bash
-WORKDIR /home/theia
+WORKDIR /home/theia/project
 COPY --from=0 /home/theia /home/theia
 EXPOSE 3000
 ENV SHELL /bin/bash
 ENV USE_LOCAL_GIT true
-ENTRYPOINT [ "yarn", "theia", "start", "/home/project", "--hostname=0.0.0.0" ]
+USER theia
+ENTRYPOINT [ "yarn", "theia", "start", "/home/theia/project", "--hostname=0.0.0.0" ]
```

Comment:
Is there a way to preserve `/home/project` that we don't need to updated repo, images descriptions, tweets and so on?

Reply:
`mkdir home/project`? or i am missing something

Reply:
Will add that instead of doing it on the default user folder